prov/rxm/src/rxm.h

```diff
@@ -260,6 +261,7 @@ struct rxm_tx_entry {
 	void *context;
 	uint64_t flags;
 	uint64_t comp_flags;
+	struct dlist_entry postponed_entry;
```

Comment:
Can this be moved into an unnamed union with struct fi_context at the start of the structure?

Reply:
Hmm, I'm not sure. If a core provider uses this fi_context for its own needs (modifies `internal[4]` array), it might be an issue (it will cause that the `postponed_entry` will be changed)

Reply:
While the request is on the postponed list, it won't be posted to underlying provider.  Once it is posted, it's no longer postponed.  The usages don't appear to overlap.

Reply:
Opps, Thanks. Good catch 👍

Reply:
I've updated structure as you requested